framework/flamingo/sessions.go

```diff
@@ -76,6 +79,10 @@ func (m *SessionModule) Inject(config *struct {
 	m.redisClusterMode = config.RedisClusterMode
 	m.healthcheckSession = config.CheckSession
 
+	if strings.TrimSpace(config.RedisKeyPrefix) != "" {
+		m.redisKeyPrefix = &config.RedisKeyPrefix
```

Comment:
@mkrill We should also trim the value here before assigning it.

Reply:
Done. Might also make sense for other parameters in the future, probably.